cucumber/policy/features/support/world.rb

```diff
@@ -26,8 +26,8 @@ def invoke status: :ok, &block
         puts result if @echo
       end
     rescue RestClient::Exception => e
-      @exception = e
-      raise e unless status == e.http_code
+      expect(e.http_code).to eq(status) if status.is_a? Integer
```

Comment:
`status` should always be the same data type, so I recommend we remove the `if` test.

Reply:
Actually, status is a symbol by default - so this update will only check the http_code against the status if a numeric value has been passed to the `invoke` function (which appears to be how it's used from reviewing the other cukes)

Reply:
The alternative would be converting `status` within `invoke` by calling something like `status = Rack::Utils::SYMBOL_TO_STATUS_CODE[status] if !status.is_a? Integer` to make sure the comparison always works as expected.

Reply:
In my view, numerical status codes are a better choice because that's what's specified in the HTTP spec. The symbols is more of a Rails-ism, so somebody using the cucumber tests as a reference who's not a Ruby programmer will be familiar with the numerical codes but not the symbols.

Reply:
I would like to see the tests use numerical status codes, because codes are given by the HTTP spec, and also to be consistent with other tests such as https://relishapp.com/conjur/possum-api/docs/rbac-privileges-control-whether-a-role-can-update-and-or-fetch-a-secret

Reply:
I am implementing this, but it is not as pretty as I'd like.

I want the status codes to remain optional, because I am calling invoke from some generic step definitions that don't assume what the output is going to be (and it could be successful or not, so I don't want to set the default to `success`).

This is leading invoke to look like this:
```
def invoke status: nil, &block
    begin
      @result = yield
      raise "Expected invocation to be denied" if status && status != 200
      @result.tap do |result|
        puts result if @echo
      end
    rescue RestClient::Exception => e
      expect(e.http_code).to eq(status) if status
      @result = e.response.body
    end
  end
```

At this point I just really want the status code check to live someplace else. We already have a "the error code is [code]" step definition - why not just update the cukes that are passing invoke a status code to use this step definition?